overlord/hookstate/ctlcmd/helpers.go

```diff
@@ -132,7 +132,7 @@ func queueCommand(context *hookstate.Context, tts []*state.TaskSet) error {
 func runServiceCommand(context *hookstate.Context, inst *servicestate.Instruction) error {
 	if context == nil {
 		// this message is reused in health.go
-		return fmt.Errorf(i18n.G("cannot %s without a context"), inst.Action)
+		return fmt.Errorf(i18n.G(`The snapctl %s action can only be invoked from within a snap`), inst.Action)
```

Comment:
this should use the same message as well but command name here is inst.Action

Reply:
@mardy was this part addressed?

Reply:
Yes, it now uses the same error message as in other parts: https://github.com/mardy/snapd/blob/snapctl-context-err/overlord/hookstate/ctlcmd/helpers.go#L133-L135

(if you are merging this yourself please remember to squash :-) )